ServerCore/Components/NotificationComponent.razor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace ServerCore.Components
+{
+    public partial class NotificationComponent : IDisposable
+    {
+        [Parameter]
+        public int EventID { get; set; }
+
+        [Parameter]
+        public int TeamID { get; set; }
+
+        [Parameter]
+        public int PlayerID { get; set; }
+
+        [Inject]
+        protected IJSRuntime JS { get; set; } = default!;
+
+        private IDisposable listener = null;
+
+        public NotificationComponent()
+        {
+        }
+
+        protected override void OnInitialized()
+        {
+            this.listener = NotificationHelper.RegisterForNotifications(EventID, TeamID, PlayerID, this.OnNotification);
+            base.OnInitialized();
+        }
+
+        private void OnNotification(object sender, NotificationEventArgs args)
+        {
+            this.JS.InvokeVoidAsync("displayNotification", args.Notification);
```

Comment:
If not awaiting here is intentional, please put in a discard so we don't get a warning and to show it's intentional

Reply:
What is a "discard"?

Reply:
Found it, nm